Added reviewers invitation template (#28)
pythonindia/pyconindia-handbook: templates/emails/reviewers_group_invitation.rst

Hello,
We hope you are in good health and doing well.

> This email is in regard to the PyCon India 2020 review team. PyCon India is going virtual this year considering the current situations (https://twitter.com/pyconindia/status/1253732245635952643). We, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and more accessible to our esteemed attendees from all over the world in more than one way.

Reviewer: ```suggestion
This email is in regard to the PyCon India 2020 review team. PyCon India is going virtual this year considering the current situations (https://twitter.com/pyconindia/status/1253732245635952643). We, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and more accessible to our esteemed attendees from all over the world in ways more than one.
```

> Hence, we would like to put push towards bringing up some quality talks to be presented this  Pycon 2020. We are creating a team to review the talk proposals that should be presented in Pycon. Since this is a peer-review process and driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience in Python and/or with Python community will be a core part of this team we intend to build going ahead. 

Reviewer: ```suggestion
Hence, we would like to put push towards bringing up some quality talks to be presented this PyCon India 2020. We are creating a team to review the talk proposals that should be presented in PyCon India. Since this is a peer-review process and driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience in Python and/or with Python community will be a core part of this team we intend to build going ahead. 
```

> Your role will be mainly to review the talk proposals submitted in your area of expertise. We would like you to get your feedback to find a fantastic talk lineup for PyCon India 2020. 

Reviewer: ```suggestion

Your role will be to review the talk proposals submitted in your area of expertise and provide feedback on the same to build a fantastic talk lineup for PyCon India 2020. 
```

> If you think you can help us with this, please reply to the email confirming and we will add you to the group where we will discuss and plan/schedule sessions. 

Reviewer: ```suggestion
If you think you can help us with this, please reply to the email confirming your participation and we will add you to the group where we will discuss and plan/schedule sessions. 
```

Hello,

> We hope you are in good health and doing well.

Reviewer: Not sure why the linebreak here might look odd on email.

Author: Yeah will remove that.

> This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.

Reviewer: reviewer group --> review team might sound better.
Please change Pycon to `PyCon India` everywhere with C capital

```suggestion
This email is in regard to the PyCon India 2020 review team. Pycon India is going virtual this year considering the current situations (LINK the announcement tweet). We (conference organizers) take this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and more accessible to our esteemed attendees from all over the world in ways more than one.
```

Reviewer: Please add/modify any changes I suggested above into your text, as you like.

> This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.

Reviewer 2: ```suggestion
This email is in regard to PyCon India 2020 reviewer group. Pycon India is going virtual this year, considering the current situations. We, the conference organizers, are taking the opportunity to make this conference a great experience for attendees in other innovative ways.
```

> We would like to put some more effort in bringing up some amazing talks to be presented this  Pycon 2020. We are creating a team of volunteers to review the talk proposals that should be presented in Pycon. Since this is a peer review process and driven by volunteers, we are looking for more volunteers to join us and help in making this a success. We believe you have great experience in Python and/or Python community and we would love if you can be a part of this group.

Reviewer: ```suggestion
Hence, we would like to put push towards bringing up some quality talks to be presented this  Pycon 2020. We are creating a team to review the talk proposals that should be presented in Pycon. Since this is a peer-review process and driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience in Python and/or with Python community will be a core part of this team we intend to build going ahead. 
```

Make it personal to them for a better reach.

Reviewer 2: ```suggestion
Hence, we would like to push towards bringing in quality talks to be presented this PyCon 2020. We are creating a team to review the talk proposals that should be presented in PyCon. Since the talks are peer-reviewed and the process is driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience with Python and the Python community will add vast value to the team we intend to build going ahead.
```

> Your role will be mainly to review the content of the talks submitted. You will be volunteering for a few talks you see fit as per your skills (we will help you with this) and provide your feedback if this should be a part of Pycon 2020. 

Reviewer: ```suggestion
Your role will be mainly to review th talk proposals submitted. You will be volunteering for a few talks you see fit as per your skills (we will help you with this) and provide feedback if those talks could be the right fit for Pycon 2020. 
```

I don't get what this means, `You will be volunteering for a few talks you see fit as per your skills (we will help you with this)`

Reviewer 2: ```suggestion
Your role will be mainly to review the talks submitted in your area of expertise. We would like you to get your feedback to find a fantastic talk lineup for PyCon India 2020.
```

The second line can become very confusing. Hence, it is better to refactor.

> If you think you can help us with this, please reply to the email confirming and we will add you to the group where we will discuss and plan/schedule sessions. 
> FAQs
> 1. How much time do I need to dedicate ?
> Ans: For review, it will be around 10-15 hours over a period of 20 - 25 days

Reviewer: Have to let them know they are needed and are extremely essential to process.

```suggestion
If you feel you can help and play an essential role in growing our community's beloved conference, please reply to the email confirming your participation and we will add you to the group where we can further discuss and plan/schedule sessions. 

FAQs

1. How much time would I need to commit?
For review, it will be around 10-15 hours over a period of 20 - 25 days
```

Wouldn't the time be more, I mean I am no reviewer. PyCon does get a lot of proposals + we have this new video feature. @sayanchowdhury what do you think

Author: Got that details from ritesh, Yes a confirmation would be nice.

Author: Confirmed with the team we will go with buffer of 20 hours.

2. How many meetings do I need to attend?

> Ans: For review, you will be required to attend 4-5 meetings for general discussion around review.

Reviewer 2: ```suggestion
Ans: You will be required to attend 4-5 meetings for discussions around the review process.
```

> Thank you.

Reviewer: I feel we can end with a much warmer note 

- If you have any question feel free to ask or something 
- Looking forward to hearing back from you 
- A personal touch would go a long way.

Author: " Hoping to see you onboard with the team. Feel free to reach out for any queries " How does this sound. I want to keep it minimal.

Reviewer: Minimal works :tada:
